locations/spiders/dominos_pizza.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import re
+
+from locations.items import GeojsonPointItem
+ALLOWED_ZIPS=[range(500,9899),range(10000,21299),range(21400,26899),range(27000,34299),range(21400,26899),range(27000,34299),
```

Comment:
Instead of searching a bunch of zipcodes, please use the directory of all stores in the US on their site here: https://pizza.dominos.com/.